src/julia.h

```diff
@@ -72,6 +72,7 @@ typedef struct _jl_tls_states_t *jl_ptls_t;
 #include "uv.h"
 #endif
 #include "gc-interface.h"
+#include "gc-common.h"
```

Comment:
This doesn't work as `gc-common.h` in fact also tries to `#include "julia.h"`.... specifically it needs various typedefs that are only declared later in this file.

(The Julia kernel header files really are quite a tangled up mess at this point, unfortunately)